addOns/oast/src/main/resources/org/zaproxy/addon/oast/resources/Messages.properties

```diff
@@ -60,6 +60,10 @@ oast.panel.table.column.referer=Referer
 oast.panel.currentState.tooltip.lastPolled = {0}: Last Polled Time
 oast.panel.currentState.lastPoll= {0}: {1}
 
+oast.popup.menu.insertPayload=Insert Payload
```

Comment:
Since this can be shown pretty much anywhere we should be more clear, e.g. `Insert OAST Payload`.

Reply:
okay working on these tasks.